contracts/lib/MiMc7.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.5.0;
```

Comment:
Where did this come from?  Mimc7 is outdated.

Reply:
This repo was outdated, and Mimc7 was in the private/internal smart contracts with most of the rest of the contracts to publish identity roots/states.
On the first commit of this PR was added as general import of the last version of the smart contracts (that was in the private repo)
This is the commit that imports it together with the rest of contracts that were outdated in this repo: https://github.com/iden3/contracts/commit/27e36bc52540a6ca0a2c26fd06e8c253b99d7995